db/column_family.h

```diff
@@ -222,7 +222,7 @@ struct SuperVersion {
   // Cleanup unrefs mem, imm and current. Also, it stores all memtables
   // that needs to be deleted in to_delete vector. Unrefing those
   // objects needs to be done in the mutex
-  void Cleanup();
+  void Cleanup(const bool fromMemPurge = false);
```

Comment:
From Google C++ style:

  * For a function parameter passed by value, const has no effect on the caller, thus is not recommended in function declarations. See [TotW \#109](https://abseil.io/tips/109).

Reply:
This website is a gold mine, thanks for the reference. I will remove the const keyword from the function declaration in the next commit.